erasmo07/personal-page
# Added: New App call blog with it add all template

## Status
**READY**

## Migrations
YES

## Description
It add all relate with post such as: template, models, admin
view and others.

For that reazon add on .gitignore migrations.

Change the view that shows the main page to a view base on classes
this is to send the last Post.

## Deploy Notes
```sh
git pull origin master
python manage.py makemigrations
python manage.py migrate
touch uwsgi.ini
```

## Steps to Test or Reproduce
    # For show blog section.
        1. Go to blog sección on main page.
        2. Click on "See blog".

    # For create one post.
        1. Go to admin panel
        2. Click on posts.
        3. Click on "Add new post."
